@kiva/kv-shop/package.json

```diff
@@ -0,0 +1,53 @@
+{
+    "name": "@kiva/kv-shop",
+    "version": "1.0.0",
+    "type": "module",
+    "publishConfig": {
+        "access": "public"
+    },
+    "main": "dist/index.cjs",
+    "module": "dist/index.js",
+    "types": "dist/index.d.ts",
+    "files": [
+        "dist"
+    ],
+    "exports": {
+        ".": {
+            "require": "./dist/index.cjs",
+            "import": "./dist/index.js",
+            "types": "./dist/index.d.ts"
+        }
+    },
+    "scripts": {
+        "build": "tsup src/**/*.ts --format cjs,esm --dts --clean && cp -R src/components/ dist/components/",
+        "lint": "eslint --ext .ts,.vue ./src",
+        "test": "echo No test yet for kv-shop"
+    },
+    "devDependencies": {
+        "@typescript-eslint/eslint-plugin": "^5.59.7",
+        "@typescript-eslint/parser": "^5.59.7",
+        "@vue/eslint-config-typescript": "^11.0.3",
+        "eslint-plugin-vue": "^9.14.0",
+        "tsup": "^6.7.0",
+        "typescript": "^5.0.4",
+        "vue": "2.6",
+        "vue-eslint-parser": "^9.3.0"
+    },
+    "dependencies": {
+        "@apollo/client": "^3.7.14",
+        "@kiva/kv-components": "^3.19.0",
+        "@types/braintree-web-drop-in": "^1.34.2",
```

Comment:
This can be a dev dep

Reply:
I tried that initially but ran into errors saying ti should be listed in the dependencies. Maybe that was a temporary issue though?